docs/kubernetes/kctlr-f5-resource.rst

```diff
@@ -0,0 +1,114 @@
+- HTTP virtual servers that needs customization beyond the |kctlr| `Ingress annotations`_ (like iApps);
+- virtual servers for TCP or UDP applications that require non-HTTP ingress.
+
+
+.. _f5 resource properties:
+
+F5 Resource properties
+----------------------
+
+The F5 resource is a JSON blob consisting of the properties shown in the table below.
+
+
+.. table:: F5 Resource properties
+
+   ======================= ======================================================== ===========
+   Property                Description                                              Required
+   ======================= ======================================================== ===========
+   data                    A JSON object                                            Required
+   ----------------------- -------------------------------------------------------- -----------
+   - frontend              Defines BIG-IP objects.
+   ----------------------- -------------------------------------------------------- -----------
+   - backend               Identifies the Service you want to proxy.
+
+                           Defines a BIG-IP health monitor(s) for the Service.
+   ----------------------- -------------------------------------------------------- -----------
+   f5type                  A :code:`label` property watched by the |kctlr|.         Required
+   ----------------------- -------------------------------------------------------- -----------
+   schema                  Tells the |kctlr| how to interpret the encoded data.     Required
+   ======================= ======================================================== ===========
+
+.. _f5 resource frontend:
+
+Data.Frontend
+`````````````
+
+The :code:`data.frontend` property defines the objects you want to create on the BIG-IP system for a specific Service.
+In this section, you can :ref:`define a custom virtual server <kctlr-per-svc-vs>` using the :code:`virtualServer` properties or :ref:`deploy an iApp <kctlr-deploy-iapps>` using the :code:`iApp` properties.
+
+- `k8s-bigip-ctlr virtual server configuration parameters`_
+- `k8s-bigip-ctlr iApp configuration parameters`_
+
+The iApp configuration parameters include a set of customizable ``iappVariables`` parameters. These custom user-defined parameters must correspond to fields in the iApp template you want to launch. You can also define the `iApp pool member table`_ that the iApp creates on the BIG-IP system.
+
+.. tip:: With the iApp configuration parameters, you can also deploy objects across BIG-IP modules (for example, ASM and LTM).
+
+.. _f5 resource backend:
+
+Data.Backend
+````````````
+
+Use the :code:`data.backend` property to identify the `Kubernetes Service`_ that makes up the back end server pool.
+
+.. tip:: You can also define BIG-IP health monitors in this section.
+
+
+F5 type
+```````
+
+The usage of the :code:`f5type` property differs depending on whether you're using it in Kubernetes or OpenShift.
```

Comment:
We should use the kubernetes term here, the f5type is a label in the resource metadata.